.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.19
+          go-version-file: go.mod
```

Comment:
Are quotes not required around go.mod?

Reply:
they are optional, i will still add them as to maintain the code style